src/assets/plunker/index.html

```diff
@@ -23,10 +23,9 @@
   </script>
 
   <!-- Load the Angular Material 2 stylesheet -->
-  <link href="https://unpkg.com/@angular/material/core/theming/prebuilt/indigo-pink.css" rel="stylesheet">
-
+  <link href="https://cdn.rawgit.com/angular/material2-builds/master/core/theming/prebuilt/indigo-pink.css" rel="stylesheet">
```

Comment:
Using the CDN for rawgit when loading the `master` branch may cause issues.

See https://github.com/rgrove/rawgit/issues/51